cmd/apiserver/app/server/run_server.go

```diff
@@ -42,7 +42,8 @@ func RunServer(opts *ServiceCatalogServerOptions) error {
 func runTPRServer(opts *ServiceCatalogServerOptions) error {
 	tprOpts := opts.TPROptions
 	glog.Infoln("Installing TPR types to the cluster")
-	if err := tpr.InstallTypes(tprOpts.clIface); err != nil {
+	tprInstaller := tpr.NewInstaller(tprOpts.clIface.Extensions().ThirdPartyResources())
```

Comment:
Sorry, I don't mean having an exported func in the package will make things 'untestable'. 
It was more if anyone needs to use that go package, most likely they will look for a provided mock for testing, or they will make one themselves. Mocking an object with an interface is more resilient as the codebase, the interface is enforced by the compiler.

We do need to mock TPRInstaller for the unit test of `run_server.go`, so imo I believe it is better to start off the package as an object.